system/objects.rst

```diff
@@ -88,15 +88,97 @@ to Zammad. If you made a mistake or just want to discard your changes, click
 System attributes
 -----------------
 
-Zammad comes with pre-configured attributes. Some of these currently do not
-provide the possibility to edit them via UI (or at all).
+Zammad comes with pre-configured attributes. Some of them can't be edited via
+UI (or at all).
 
 This is not a bug but is to save you from possibly nuking Zammad.
```

Comment:
How about rephrasing: `This is required for proper operation of Zammad and not a bug.` or similar. I would also connect it to the previous paragraph, it doesn't need an own one.